app/views/catalog/_catalog_listing.html.erb

```diff
@@ -4,14 +4,13 @@
       <%= link_to category.name, category_equipment_models_path(category) %>
     </h2>
     <div class="thumbnails row">
-      <% equipment_models.each do |equipment_model| %>
+      <% (equipment_models[0, 3]).each do |equipment_model| %>
```

Comment:
Hmm, I might be missing something, but why are we looking at only a subset of the equipment models?